src/components/DropDown/DropDown.js

```diff
@@ -185,17 +188,19 @@ const DropDown = React.memo(function DropDown({
 DropDown.propTypes = {
   header: PropTypes.node,
   items: PropTypes.arrayOf(PropTypes.node).isRequired,
-  label: PropTypes.string,
+  placeholder: PropTypes.string,
```

Comment:
Should `placeholder` be any node? (so we can have style variants like mini badges)

```suggestion
  placeholder: PropTypes.node,
```